main/src/cgeo/geocaching/activity/AbstractActivity.java

```diff
@@ -179,7 +181,7 @@ protected void buildDetailsContextMenu(final ActionMode actionMode, final Menu m
         menu.findItem(R.id.menu_translate_to_english).setVisible(!copyOnly && !localeIsEnglish);
     }
 
-    protected boolean onClipboardItemSelected(@NonNull final ActionMode actionMode, final MenuItem item, final CharSequence clickedItemText) {
+    protected boolean onClipboardItemSelected(@NonNull final ActionMode actionMode, final MenuItem item, final CharSequence clickedItemText, final Geocache cache) {
```

Comment:
Would you mind making the added parameter `@Nullable` so that we know from the prototype that `null` is an acceptable argument?